11ty/definitions/boogaloo.md

```diff
@@ -0,0 +1,46 @@
+  - text: The Boogaloo Movement Is Not What You Think
+    href: https://www.bellingcat.com/news/2020/05/27/the-boogaloo-movement-is-not-what-you-think/
+---
+
+
+original meaning: A type of music made up of a mix of mambo, cha cha, pachanga, doo-wop and soul. "Boogaloo" has since been appropriated by Alt-Right and White Supremacists groups to refer to a race-based civil war. Other terms for Boogaloo include: _boog, boojahideen, big igloo, and big luau_.
+
+The co-opted use of “Boogaloo” is a [dog-whistle](https://en.m.wikipedia.org/wiki/Dog-whistle_politics), used to communicate about, and signal support for militias inciting violence to escalate and accelerate violent uprisings against the government. Part of electing to use this term, as well as Hawaiian shirts and igloos is to avoid existing automated content flags used on social media sites.
+
+Its usage is derived from memes about the film _Breakin’ 2: Electric Boogaloo_, riffing on the title – notably, as “Civil War 2: Electric Boogaloo”. Appending “Electric Boogaloo” to other concepts as a joke was born in and largely popularized by early internet message boards such as Something Awful and 4chan.
+
+Some Alt-Right members self-identify as "Boogaloo Boys" (or "Bois"), or speak of "preparing / showing up for the Boogaloo". Many dress in Hawaiian shirts and/or military wear to distinguish themselves at protests and gatherings. One effect of wearing the Hawaiian shirt is to co-opt the public’s existing notions about Hawaiian culture with peaceful, impartial observation, and not radical militarism.
+
+## Issues
+
+Similarly to the [OK Hand symbol](/definitions/ok-hand.md), the word itself does not have racist origins (and in fact, is a product of the Latin music community) and neither does the original meme (a joke about movie sequels). However, language evolves, and "Boogaloo" has become a term used quite seriously by Alt-Right and White Supremacist communities.
+
+It should be noted that this approach of indoctrinating symbols of hate is one long-documented, and [used for the Swastika as well](https://theconversation.com/how-nazis-twisted-the-swastika-into-a-symbol-of-hate-83020)
+
+## Related symbols
+
+Both the Hawaiian shirt and Igloo snow huts are associated with Indigenous cultures. Before being co-opted by the Boogaloo movement, both symbols were co-opted by American popular culture.
+
+Hawaiian shirts were popularized by the Tiki culture movement, created after American military returned from being stationed in the South Pacific during World War II. Due to a lack of including and elevating Polynesian people in the movement, it is considered to be both cultural appropriation and colonial nostalgia.
+
+## Impact
+
+Although this was previously a non-violent / non-hateful meme, continuing to use it supports the plausible deniability of those who do use it with serious intent – and may put you in danger of being mistaken for a "Boogaloo Boy".
```

Comment:
I think the impact here is a bit solely focused on perpetrator and not the harmed.

One aspect of hate symbols being dogwhistled is that those who are most impacted/vulnerable will not know what your intent is and therefore will feel the fear regardless.